test/test_lp.jl

```diff
@@ -10,76 +11,76 @@ TOL = 1e-3
     p = minimize(abs(x), x<=-1)
     @test vexity(p) == ConvexVexity()
     solve!(p)
-    @test isapprox(p.optval, 1, atol=TOL)
-    @test isapprox(evaluate(abs(x)), 1, atol=TOL)
+    @test p.optval ≈ 1 atol=TOL
+    @test evaluate(abs(x)) ≈ 1 atol=TOL
 
     x = Variable(2,2)
     p = minimize(sum(abs(x)), x[2,2]>=1, x[1,1]>=1, x>=0)
     @test vexity(p) == ConvexVexity()
     solve!(p)
-    @test isapprox(p.optval, 2, atol=TOL)
-    @test isapprox(evaluate(sum(abs(x))), 2, atol=TOL)
+    @test p.optval ≈ 2 atol=TOL
+    @test evaluate(sum(abs(x))) ≈ 2 atol=TOL
   end
 
   @testset "maximum atom" begin
     x = Variable(10)
-    a = rand(10, 1)
+    a = shuffle(collect(0.1:0.1:1.0))
     p = minimize(maximum(x), x >= a)
     @test vexity(p) == ConvexVexity()
     solve!(p)
-    @test isapprox(p.optval, maximum(a), atol=TOL)
-    @test isapprox(evaluate(maximum(x)), maximum(a), atol=TOL)
+    @test p.optval ≈ maximum(a) atol=TOL
+    @test evaluate(maximum(x)) ≈ maximum(a) atol=TOL
   end
 
   @testset "minimum atom" begin
     x = Variable(1)
-    a = rand(10, 10)
+    a = reshape(shuffle(collect(0.01:0.01:1.0)), (10, 10))
     p = maximize(minimum(x), x <= a)
     @test vexity(p) == ConvexVexity()
     solve!(p)
-    @test isapprox(p.optval, minimum(a), atol=TOL)
-    @test isapprox(evaluate(minimum(x)), minimum(a), atol=TOL)
+    @test p.optval ≈ minimum(a) atol=TOL
+    @test evaluate(minimum(x)) ≈ minimum(a) atol=TOL
 
     x = Variable(4, 4)
     y = Variable(4, 6)
     z = Variable(1)
     c = ones(4, 1)
-    d = 2 * ones(6, 1)
+    d = fill(2.0, (6, 1))
     constraints = [[x y] <= 2, z <= 0, z <= x, 2z >= -1]
     objective = sum(x + z) + minimum(y) + c' * y * d
     p = maximize(objective, constraints)
     @test vexity(p) == ConvexVexity()
     solve!(p)
-    @test isapprox(p.optval, 130, atol=TOL)
-    @test isapprox((evaluate(objective))[1], 130, atol=TOL)
+    @test p.optval ≈ 130 atol=TOL
+    @test (evaluate(objective))[1] ≈ 130 atol=TOL
   end
 
   @testset "max atom" begin
     x = Variable(10, 10)
     y = Variable(10, 10)
-    a = rand(10, 10)
-    b = rand(10, 10)
+    a = reshape(shuffle(collect(0.01:0.01:1.0)), (10, 10))
```

Comment:
(Not that it matters, but) why this change?

Reply:
See https://github.com/invenia/Convex.jl/pull/1#discussion_r229035088

Reply:
This fixed an issue that was tricky to track down. @molet noted in https://github.com/invenia/Convex.jl/pull/1#discussion_r229035088:

> Using `rand` led to test failures with SCS solver for some cases. I think it happened because for some set of random numbers the problem might not be well conditioned (i.e. some numbers are too close to each other). So I changed it to a set of numbers from an equidistant grid but to still put some randomness there I shuffled them.